tests/functional.bats

```diff
@@ -42,7 +42,7 @@ load test_helper
 @test "(cmd) invalid dokku commands receive standard response" {
   daemon_start
 
-  run client_command "app"
+  run client_command "batman"
```

Comment:
`app` is also not a valid command, was this necessary?

Reply:
i think this one can be chalked up to a bout of confusion on my part late at night. i'll revert this and verify tests pass.